src/figuresClasses.ts

```diff
@@ -1,11 +1,98 @@
-export interface Figure {}
+interface Figure {
+  getArea(): number;
+  getInfo(): string;
+  shape: string;
+  color: string;
+}
+
+export class Triangle implements Figure {
+  public shape = 'triangle';
+  public color: string;
```

Comment:
```suggestion
  public shape = 'triangle';

  public color: string;
```
to fix lint error
read all errors and fix them